corehq/apps/hqcase/management/commands/ptop_preindex.py

```diff
@@ -33,7 +33,12 @@ def get_reindex_commands(alias_name):
         'hqgroups': [('ptop_reindexer_v2', {'index': 'group'})],
         'report_xforms': ['ptop_fast_reindex_reportxforms'],
         'report_cases': ['ptop_fast_reindex_reportcases'],
-        'case_search': [('ptop_reindexer_v2', {'index': 'case-search'})]
+        'case_search': [('ptop_reindexer_v2', {'index': 'case-search'})],
+        'ledgers': [
+            ('ptop_reindexer_v2', {'index': 'ledger-v1'}),
+            ('ptop_reindexer_v2', {'index': 'ledger-v2'}),
+        ],
```

Comment:
will be interesting to see how these do. probably the biggest reindex to date on the new stuff